recipe/meta.yaml

```diff
@@ -1,26 +1,27 @@
 {% set name = "youtube_dl" %}
-{% set version = "2021.6.6" %}
+{% set version = "2021.12.17" %}
 
 package:
   name: youtube-dl
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/y/youtube_dl/youtube_dl-{{ version }}.tar.gz
-  sha256: cb2d3ee002158ede783e97a82c95f3817594df54367ea6a77ce5ceea4772f0ab
+  sha256: bc59e86c5d15d887ac590454511f08ce2c47698d5a82c27bfe27b5d814bbaed2
 
 build:
-  number: 1
+  number: 0
+  noarch: python
```

Comment:
@nehaljwani do you remember why we remove noarch? We had it before if I recall correctly.

Reply:
It was removed in this commit:

https://github.com/conda-forge/youtube-dl-feedstock/commit/b79ca914979fff0f3931f162eb24c8a27884d69c#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aL16

I don't remember the reason!

Reply:
Well, I did it and I do not remember :smile: 

Let's give it a try and we revert if things break.